main/src/ca/uwaterloo/flix/language/phase/jvm/BackendObjType.scala

```diff
@@ -1608,73 +1607,41 @@ object BackendObjType {
   // Java Types
   //
 
-  case object String extends BackendObjType {
+  object String {
 
-    def JoinMethod: StaticMethod = StaticMethod(this.jvmName,
-      "join", mkDescriptor(CharSequence.toTpe, BackendType.Array(CharSequence.toTpe))(String.toTpe))
-
-    def BoolValueOf: StaticMethod = StaticMethod(this.jvmName,
-      "valueOf", mkDescriptor(BackendType.Bool)(this.toTpe))
-
-    def CharValueOf: StaticMethod = StaticMethod(this.jvmName,
-      "valueOf", mkDescriptor(BackendType.Char)(this.toTpe))
-
-    // implicit use of Int8 as Int32
-    def Int8ValueOf: StaticMethod = StaticMethod(this.jvmName,
-      "valueOf", mkDescriptor(BackendType.Int32)(this.toTpe))
-
-    // implicit use of Int16 as Int32
-    def Int16ValueOf: StaticMethod = StaticMethod(this.jvmName,
-      "valueOf", mkDescriptor(BackendType.Int32)(this.toTpe))
-
-    def Int32ValueOf: StaticMethod = StaticMethod(this.jvmName,
-      "valueOf", mkDescriptor(BackendType.Int32)(this.toTpe))
-
-    def Int64ValueOf: StaticMethod = StaticMethod(this.jvmName,
-      "valueOf", mkDescriptor(BackendType.Int64)(this.toTpe))
-
-    def Float32ValueOf: StaticMethod = StaticMethod(this.jvmName,
-      "valueOf", mkDescriptor(BackendType.Float32)(this.toTpe))
-
-    def Float64ValueOf: StaticMethod = StaticMethod(this.jvmName,
-      "valueOf", mkDescriptor(BackendType.Float64)(this.toTpe))
-
-    def ObjectValueOf: StaticMethod = StaticMethod(this.jvmName,
-      "valueOf", mkDescriptor(BackendObjType.JavaObject.toTpe)(this.toTpe))
-
-    def Concat: InstanceMethod = InstanceMethod(this.jvmName,
-      "concat", mkDescriptor(this.toTpe)(this.toTpe))
+    def Concat: InstanceMethod =
```

Comment:
I didn't fully remove this object since Concat is used multiple places, but String doesn't implement BOT so this is just to hold this method name/descriptor